fastlane/metadata/android/hu-HU/full_description.txt

```diff
@@ -1,43 +1,42 @@
+Az Element egy biztonságos üzenetküldő, és egy csapatmunka app, amely távoli munkavégzéshez is alkalmas lehet. Az alkalmazás végponti titkosítás használatával biztosít videó konferencia, fájlmegosztás, és audio hívás lehetőségeket.
 
-<b>Element tulajdonságai:</b>
-- Fejlett online kommunikációs eszköz
-- Teljesen titkosított üzenetküldés biztonságos céges kommunikációt kínál még a távdolgozóknak is
-- Elosztott csevegés a Matrix nyílt forráskódú keretrendszer felhasználásával
-- Bizontságos fájl megosztás titkosítottan projektek kezeléséhez
-- Videó hívás VoIP-pal és képernyőmegosztással
-- Könnyen integrálható a kedvenc online kollaborációs eszközöddel, projekt menedzsment eszközzel, VoIP szolgáltatással vagy más csoport üzenetküldő alkalmazással
+<b>Az Element funkciói többek között:</b>
+- Fejlett online kommunikációs eszközök
+- Titkosított üzenetek a biztonságos céges kommunikációhoz, otthonról dolgozóknak is
+- Decentralizált chat a nyílt forráskódú Matrix protokoll használatával
+- Biztonságos fájlmegosztáss a projektek kezeléséhez
+- Videochat, VoIP, és képernyőmegosztási lehetőséggel 
+- Egyszerű integráció a kedvenc online kollaborációs eszközeiddel, projektkezelési eszközökkel, VoIP szolgáltatásokkal, és más csoportos üzenetküldő alkalmazásokkal
 
-Element teljesen más mint a többi üzenetküldő alkalmazás. Matrixot használ, egy nyílt hálózatot a decentralizált biztonságos kommunikációhoz. Lehetőséget ad saját szerver üzemeltetésére ami maximális tulajdont és kontrollt biztosít az adatok fölött.
+Element is completely different from other messaging and collaboration apps. It operates on Matrix, an open network for secure messaging and decentralized communication. It allows self-hosting to give users maximum ownership and control of their data and messages.
 
-<b>Magánélet védelme és titkosított üzenetküldés</b>
-Element megóv a kéretlen hirdetésektől, adatbányászattól és a különböző szigetszerű megoldásoktól. Minden adatot biztonságba helyez, egy az egybe videó és hang kommunikáció végpontok között titkosítva ahol az eszközök hitelesítve vannak.
+<b>Privacy and encrypted messaging</b>
+Element protects you from unwanted ads, data mining and walled gardens. It also secures all your data, one-to-one video and voice communication through end-to-end encryption and cross-signed device verification.
 
-Element a kezedbe adja az adatvédelmi irányítást miközben bárkivel kommunikálhatsz a Matrix hálózatban vagy más üzleti kollaborációs eszközzel ami integrálva van, mint amilyen a Slack.
+Element gives you control over your privacy while allowing you to communicate securely with anyone on the Matrix network, or other business collaboration tools by integrating with apps such as Slack.
 
-<b>Element futtatható saját szerveren</b>
+<b>Element can be self-hosted</b>
```

Comment:
to highlight - some translations have been reverted back to English

spoke separately with @bmarty